p2panda-rs/src/entry/tests.rs

```diff
@@ -65,6 +69,111 @@ fn sign_and_encode_roundtrip(#[case] entry: Entry, key_pair: KeyPair) {
     assert!(sign_and_encode(&entry_second, &key_pair).is_ok());
 }
 
+#[rstest]
```

Comment:
These tests are actually duplicating what we have in `SignedEntry` now that the errors are being caught there. I would say they can be removed, they don't bring anything new.